spec/rspec/rails_spec.rb

```diff
@@ -31,4 +31,26 @@
       expect(new_yaml).to eq org_yaml
     end
   end
+
+  describe "smart merge" do
+    let(:openapi_path_for_smart_merge) do
+      File.expand_path('spec/rails/doc/smart/openapi.json', repo_root)
+    end
+
+    let(:expected_path_for_smart_merge) do
+      File.expand_path('spec/rails/doc/smart/expected.json', repo_root)
```

Comment:
Can you just convert it to a normal YAML instead of a JSON?

Reply:
Done in https://github.com/k0kubun/rspec-openapi/pull/58/commits/3f9398f7186e43f49e8b926ba886846f64f92ba8